test/jasmine/tests/pie_test.js

```diff
@@ -522,33 +478,44 @@ describe('Test event property of interactions on a pie plot:', function() {
         it('should contain the correct fields', function() {
             mouseEvent('mouseover', pointPos[0], pointPos[1]);
 
-            var point0 = futureData.points[0],
-                evt = futureData.event;
-            expect(point0.originalEvent).toEqual(evt, 'points');
-            expect(evt.clientX).toEqual(pointPos[0], 'event.clientX');
-            expect(evt.clientY).toEqual(pointPos[1], 'event.clientY');
+            checkEventData(futureData);
+        });
+
+        it('should not emit a hover if you\'re dragging', function() {
+            gd._dragging = true;
+            mouseEvent('mouseover', pointPos[0], pointPos[1]);
+            expect(futureData).toBeUndefined();
+        });
+
+        it('should not emit a hover if hover is disabled', function() {
+            Plotly.relayout(gd, 'hovermode', false);
+            mouseEvent('mouseover', pointPos[0], pointPos[1]);
+            expect(futureData).toBeUndefined();
```

Comment:
This is also different from before, but consistent with other subplot types, I believe.

Reply:
Yes this is how it should be.

For the record, to get hover event data w/o visible hover labels, one should set `hoverinfo: 'none'` in all the traces of interest.